### ISSUE TITLE
how to export/serialize pywrapcp.RoutingModel ?

### ISSUE THREAD

**Issue by ihadanny** [google/or-tools#1923]
**What language and solver does this apply to?**
Python

**Describe the problem you are trying to solve.**
I'm using the routing library python wrapper. 
I would like to solve multiple parts of the model on different processes/servers, as suggested here: https://github.com/google/or-tools/issues/1213 by @lperron .
something like this:
```python
routing = pywrapcp.RoutingModel(manager, parameters)
# I would like to serialize everything here and execute the next command on a remote server
assignment = routing.SolveFromAssignmentWithParameters(initial_solution, search_parameters)
```
I didn't find any export to mps/protobuf in the pywrapcp API, what am I missing?
NOTE that on the way "back" from the solver, I did find an appropriate API: `_pywrapcp.Assignment_Save` would enable me to send the result back home to my app.

**Describe the solution you'd like**
export routingModel to something

**Describe alternatives you've considered**
I can use my own custom serializable format for describing the model...

**Additional context**
related to https://github.com/google/or-tools/issues/325

**Comment by lperron** [comment]
Unfortunately, this will not happen. We tried and it is not production
ready, and I removed the code a while back.

In practise, you have data, and a model builder that creates the routing
problem to solve.
I would be more efficient and more compact to call this builder on
decomposed data, instead of passing a fully expanded model back and forth.

Sorry.
Laurent Perron | Operations Research | lperron@google.com | (33) 1 42 68 53
00



Le mar. 17 mars 2020 à 09:15, Ido Hadanny <notifications@github.com> a
écrit :
